blockchain/testnet_actions.yaml

```diff
@@ -50,50 +49,30 @@ transfers:
 - pubKey: 114dcdeba8407fdf3e68596becec1065eaaa909a2d6bc592b3cf995bb51fde36e1ce7606ea10c0e677097351e20e97c15b0a71fc1e9f52736eafa0a0c064ed90e40ffc51d6b76b05
-  priKey: b141ba4f7e2e7b8f3bfdec5a213aab7bb17ac3e42922e0f847fe4584086debd16fdcbf00
 - pubKey: 2c1efab7f5d6621a03c95fe708ceda8150b80a8d39dfdf6a61b451c8260ac48dfc103c05d0da733dc9f88c83944dbf8765657876446695c752b1320bacb9013f57bb3a3d80c15d06
-  priKey: e8994c76512217fcfa95bb0f4a5f086a165dc0310c440ba77b401c39a53300da3482a500
 - pubKey: 9091d38072fd72611924e1e099f2e41bc612052d9369ee8a21880eb81b764043616f2f031f7a2e83b00264f6a89200268e9bf36a0495a323bfcf1cc8bf83054085098bd4059b8b02
-  priKey: bbfb39c2e16c024bbfcbd68de43960e12dd0cd521b7f290ab696691e5ea1888048896e00
 - pubKey: f4ba872ee4f7d6f353b30452bfcb55106d4496d8fad9889ab9c338f7d4f8c587106d64047feb8cfae6ab340fb037258f4184244e0047eadc4e1a836dff6b9bb0c5a1cfe34b65f804
-  priKey: 11dca42f538a9ed0cf8370bd744f278295d965312ad2a7f4263b294562696066990bfa01
 - pubKey: 7584b1e71811f904fa8f03b2bca9a50fd2110b3debceab5516a86fa332e15a768ced18076c4b15673aa6cc0ff7fa8a0152114b8f2646541d94871bd4fa646d63d30218c148663003
-  priKey: 6e295a516c34b30c824f012a529134f4c0ebc16b88e5521d14e5e7194f17d0522df95500
 - pubKey: 137b6a42dca7ecc3de0110c42cbbafb9603871f15f182fcba1b7384f7e4413b024c6aa03344cafeaf7019725d707e5ad2a564a6daa1e8d9c5ab4be2389d664b2b864966b69835e05
-  priKey: 770fc884d761de37b9da3628db96f53f99decbf76eb36df5e43a638e5ee0b1c3fa863f01
 - pubKey: 73da1252ecda320e96c0e4fe58307bcb3b1e7c063f46592d10bbcd3566a1d92f3eae8c04533abe54c937cee4b3509e008c45248edb53c81bdfafb7315fc8ea1bcef2f988eddc1502
-  priKey: 2760537cc92366a8a1bba4ce4a4f7d3b6c5cfe4c2b53b567ab04b7c74f7218fe3e8d6701
 - pubKey: bca891c113a13c4c6720e66e8c3a1a530c18ebf42097fdcc150d99c44a21bd294a664b024b8f69b4242720eb3e4f1ccd874fa3b7b8cf75631caeae5d5b3606dcbb247e0cf77fe003
-  priKey: 51eb437333708b4d6584117683aae657f3f3ef10fb2a7720fdee74a17c1cc7394e74b101
 - pubKey: 90c369a754193b1269cbd55fa90a2f254ffb2e902c53b7e4df956a0b73d1b895807885032b6c804f99383c554465b8e2173684f87995b61ccbb9b8aca9d5b0b2c417004f9a3da001
-  priKey: a48cb7eb6c4d938ad7c15694f18aa0427c5ebfbe3120d56db573cd59d0ce1503e74a3c00
 - pubKey: e8527c4787caae88f87d2f34019a927ee91c12bfdd94509bb8bf4bda02ee4e05395bad061cabd9a91aca8bfe1ad3f40699e5396e6e93d204a10b952feefd416665118ec4890b3600
-  priKey: fc9a6da640000069de182732c06fe33eef621f41a8fa4f258cdac03567f39e33ec274500
 - pubKey: 89be1f013287c8b31754528581b33c35f23aa5169a9aa6b60fd3288f582e4be10ee97a03ac49aba3b53cc4bc3b44135b0150a7684195e1882e842508633a5776c2ae50bd53d41301
-  priKey: ddc350a621db3319d86d0e679b88b0e2132f54a65066c758c3ae0adae2e4157809afc500
 subChains:
 - chainID: 2
   securityDeposit: 1000000000
   operationDeposit: 1000000000
   startHeight: 10
   parentHeightOffset: 10
+  signature: 99b8c00ae08cafc528d5b7889a62d8c2ef22354b6e7f94bac0e4ec97cfd56f15abd0650142dd349b2463cc8f9c914c6800cd7421993d01f9c1ee6d0720d3510c7c4f95b9eeaa2901
```

Comment:
This will make it difficult if we want to update the sub chain setup.

Reply:
removed